### PR TITLE
Test python 3.9 and 3.10, not 3.8 and 3.9

### PR DESCRIPTION
**Pull request by yuvipanda**
Based on
https://docs.xarray.dev/en/stable/whats-new.html#breaking-changes, xarray's latest version no longer supports Python 3.8. This causes test to fail in
https://github.com/pangeo-forge/pangeo-forge-recipes/pull/489.

So we bump up to 3.9 and 3.10!

### CODE REVIEW

**Review by cisaacstern — APPROVED** [review]
Thanks Yuvi! LGTM, and even better if we can figure out what's going on with pre-commit before merging?

**Comment by yuvipanda** [comment]
@cisaacstern  fixed in https://github.com/pangeo-forge/pangeo-forge-recipes/pull/492

**Pull request merged**
